functions/completed-checks/marking.service.js

```diff
@@ -0,0 +1,66 @@
+const moment = require('moment')
+const R = require('ramda')
+
+const sqlHelper = require('../lib/sql-helper')
+
+const markingService = {}
+
+/**
+ * Apply marking and populate answers table for the completedCheck
+ * Used in processing the psychometric report
+ * @param {Object} completedCheckMessage
+ * @param {Object} check
+ * @return {Promise<*>}
+ */
+markingService.mark = async function (completedCheckMessage, check) {
+  if (!completedCheckMessage || !completedCheckMessage.answers) {
+    throw new Error('missing or invalid argument: completed check message')
+  }
+
+  if (!check || !check.id || !check.formData) {
+    throw new Error('missing or invalid argument: check data')
+  }
+
+  const results = {
+    marks: 0,
+    maxMarks: completedCheckMessage.answers.length,
```

Comment:
I think we should definitely pick up  `maxMarks` from the form that was given to the pupil.  Ideally, the config for the pupil would be retrievable.

Reply:
@jon-shipley I changed this so that it determines the max based on the length of the form data.